### DISCUSSION TITLE
fresh install bootstrapping failed

### DISCUSSION THREAD

**Discussion by TomCJones** [cake-build/cake#3300] [Q&A]
I may have made a mistake by installing from package manager - but my path forward is unknowable!

C:\Users\rp_to>dotnet tool install --global Cake.Tool
Tool 'cake.tool' is already installed.

C:\Users\rp_to>dotnet-cake
Error: Bootstrapping failed for 'C:/Users/rp_to/build.cake'.
                                                            build.cake, line #0: Could not find script 'C:/Users/rp_to/build.cake'.

**Comment by nils-a** [comment]
@TomCJones what are you trying to do?

> bild.cake, line #0: Could not find script 'C:/Users/rp_to/build.cake'.

As a default Cake will try to use the `build.cake` file in the current directory. If that is not what you want, the first parameter to `dotnet cake` is the path to Cake file to use.

**Comment by devlead** [comment]
Do you have a file called `C:/Users/rp_to/build.cake` or where is the script you're trying to execute located.


Usage of tool is 
```bash
dotnet cake <path to cake script>
```

If script omitted it'll default to `build.cake` in current folder.
